pkg/controller/perconaservermongodb/users.go

```diff
@@ -166,8 +168,9 @@ func (r *ReconcilePerconaServerMongoDB) updateSysUsers(cr *api.PerconaServerMong
 			needRestartMongos: true, // in mgo.go:handleRsAddToShard() we use envs with this user credentials, so we need restart for update this envs
 		},
 		{
-			nameKey: envMongoDBClusterMonitorUser,
-			passKey: envMongoDBClusterMonitorPassword,
+			nameKey:        envMongoDBClusterMonitorUser,
+			passKey:        envMongoDBClusterMonitorPassword,
+			needRestartSfs: true, //need for liveness/readiness ckeck
```

Comment:
is mongos restart also needed for liveness/readyness check?

Reply:
It will be restarted after all RS